tasks/src/main/scala/dagr/tasks/bwa/Bwa.scala

```diff
@@ -61,7 +61,7 @@ object Bwa extends Configuration {
     val bwaBuffer  = new FifoBuffer[Sam].requires(memory=fifoMem)
     val altPipe    = if (doAlt) new BwaK8AltProcessor(altFile=alt) | new FifoBuffer[Sam].requires(memory=fifoMem) else Pipes.empty[Sam]
     val mergeBam   = new MergeBamAlignment(unmapped=unmappedBam, mapped=Io.StdIn, out=mappedBam, ref=ref, sortOrder=sortOrder, orientation=orientation)
-    mergeBam.requires(memory=Memory(mergeBamAlignmentMem))
+    mergeBam.requires(cores=Cores(0), memory=Memory(mergeBamAlignmentMem))
```

Comment:
I did this a while ago, so it's a little fuzzy.  Mostly I felt like mergeBam is doing very little most of the time while bwa is running, so allocating a core to it caused bwa to have to run with fewer cores.  I guess I'd rather over-subscribe a little bit here rather than under-subscribe.  I can flip it back if you'd rather.